File: 3065. Minimum Operations to Exceed Threshold Value I.py
# PYTHON
#* 3065. Minimum Operations to Exceed Threshold Value I
nums = [2,11,10,1,3]; k = 10
from typing import List
class Solution:
    def minOperations(self, nums: List[int], k: int) -> int:
        count=0
        while not all(allnum >= k for allnum in nums):
            count+=1
            nums.remove(min(nums))
        return count
    
print(Solution().minOperations(nums,k))

File: 3065. Minimum Operations to Exceed Threshold Value I.rb
#* 3065. Minimum Operations to Exceed Threshold Value I
nums = [2,11,10,1,3]; k = 10
# @param {Integer[]} nums
# @param {Integer} k
# @return {Integer}
def min_operations(nums, k)
  count=0
  until nums.all? {|all| all >= k}
    count+=1
    nums.delete_at(nums.index(nums.min))
  end
  count
end
p min_operations(nums, k)

